### DISCUSSION TITLE
TMDB Collection Operation with Smart Labels?

### DISCUSSION THREAD

**Discussion by undaunt** [meisnate12/Plex-Meta-Manager#584] [Q&A] [answered]
Is it possible to somehow apply a smart label to the TMDB collections generated via a template in a library operation? I'm trying to use that functionality, and the exclude feature, to slowly build out a lot of the smaller collections I care about automatically. However, if I cannot apply smart labels to them, it would theoretically preclude me from using smart labels across the board in lieu of Plex Collectionless. Seeing as how I think smart labels work in the collection metadata level, not library operations, I'm not sure it's currently possible.

Thanks.

**Comment by undaunt** [comment, marked as answer]
Disregard, I got it to work and then subsequently understood why the 'hide_items' flag wasn't working on the collection.. because it isn't truly a collection.

Thank you!

**Reply by meisnate12** [reply]
Yea you want every collection but tmdb collections to be smart labels
